tests/unit/SpecialResolution.spec.ts

```diff
@@ -178,6 +179,30 @@ describe('Special Resolution component', () => {
         }
       }))
 
+    // GET business documents
+    get.withArgs('businesses/CP1234567/documents')
```

Comment:
This is the original, crappy way to mock the data.

The new way is to mock the service method (no need to mock axois).

Reply:
Sorry for the extra work, but the new code looks better and means you don't have to mock axios with sinon.